Source/Common/GLTFUnarchiver.swift

```diff
@@ -35,14 +35,16 @@ public class GLTFUnarchiver {
     private var buffers: [Data?] = []
     private var materials: [SCNMaterial?] = []
     private var textures: [SCNMaterialProperty?] = []
-    private var images: [Image?] = []
+    // An array of Image or URL types
+    private var images: [Any?] = []
```

Comment:
Please note that I’m new to Swift, and my most recent experience with typed languages is TypeScript. Unfortunately I couldn’t declare this as an array of `Image | URL` types. If `Any` is unacceptable, I would consider this instead:

```suggestion
    private var images: [Image?] = []
    private var imageUrls: [URL?] = []
```